systems/subscriber/api-gateway/pb/simManager.proto

```diff
@@ -0,0 +1,140 @@
+syntax = "proto3";
```

Comment:
We should not be keeping other services proto files as this might leads to proto duplication and will make it harder to keep these files in sync or to maintain. Instead, depend directly on the other system gen package.